src/api/participantsRouter.ts

```diff
@@ -29,7 +34,10 @@ participantsRouter.get('/available', async (_req, res) => {
 
 participantsRouter.post('/', async (req, res) => {
   try {
-    const data = ParticipantCreationPartial.parse(req.body);
+    const data = {
+      ...ParticipantCreationPartial.parse(req.body),
+      status: ParticipantStatus.AwaitingApproval,
```

Comment:
It wouldn't hurt, but I don't think it really helps either. It'll mean a schema migration without really any benefit aside from this one line of code.

We'd also need to undo the default value once we add signing, because (I think) we might be creating participants in either requiresApproval or requiresSigning.

Reply:
I see, thanks for the explanation